lib/shared/sdk/usbboot/index.js

```diff
@@ -379,7 +399,7 @@ exports.scan = (options) => {
     device.adaptor = exports.name
 
     // We need to open the device in order to access _configDescriptor
-    debug(`Opening device: ${device.displayName}`)
+    debug(`Opening device: ${device.device}`)
```

Comment:
...would be useful if this debug message showed the `idPair` too ;-P

Reply:
Done!